src/govuk/components/panel/_index.scss

```diff
@@ -32,6 +40,13 @@
     margin-bottom: govuk-spacing(6);
 
     @include govuk-font($size: 48, $weight: bold);
+
+    // Decrease font size on mobile to stop long words from clipping. The clipping is most likely
+    // to happen when user increases text size on a mobile eg. using iOS Safari text resize
+    // controls.
+    @include govuk-media-query($until: tablet) {
+      font-size: 24px;
+    }
```

Comment:
I don't think there's anywhere else in the codebase we override the typography scale in this way (?), and it feels like we're running into issues with the way the helpers are currently designed to work – specifically:

- defining the tablet/desktop font size first and then the mobile feels a little 'backwards' compared to how we define our typography 'mobile first' everywhere else
- we end up with redundant CSS outputted  by the `govuk-font` mixin (the font size on mobile is defined twice)
- we're missing some of the other important things that come from the `govuk-typography-responsive` mixin (the font size in rem as well as px, and the line-height)

Maybe we need to look at making some changes to the helpers to better support this, if we're sure this is the right approach?

Reply:
@36degrees Extending the mixins sounds like something we could consider. One for the discussion list? 

How do you feel me about creating a separate card for it, if I switch these media queries to use `govuk-font` to set the font for mobile? 

Or if this whole approach needs further discussion, let's chat about it next week.

Reply:
Thanks @36degrees. Since the other fixes in this commit go a long way towards solving this issue and we haven’t come across this requirement with our other components so far, I've removed the alternative font sizes. I've added the same to the PR description and we can keep an eye out for any requests to to add this feature to `govuk-font` in the future.

Reply:
Sounds sensible!